src/pages/teachers/curriculum/index.tsx

```diff
@@ -73,9 +72,9 @@ const CurriculumHomePage: NextPage<CurriculumHomePageProps> = (props) => {
             <Hr $color={"white"} $mb={0} />
           </Box>
           <Flex $mt={[24, 80]} $mb={[80]}>
-            <HomepageCurriculumLandingHero
+            {/* <HomepageCurriculumLandingHero
```

Comment:
This is commented out and needs to be added back in once bundle issues are resolved